frontend/uddhar-frontend/src/coordinator/data/MapStat.jsx

```
@@ -0,0 +1,106 @@
+    if (areaWithPolygon) {
+      return areaWithPolygon.geojson;
+    }
+
+    // Fallback to bounding box if no polygon found
+    const areaWithBbox = data[0];
+    if (areaWithBbox?.boundingbox) {
+      const [minLat, maxLat, minLon, maxLon] = areaWithBbox.boundingbox;
+      return {
+        type: "Polygon",
+        coordinates: [[
+          [parseFloat(minLon), parseFloat(minLat)],
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)